app/presenters/publishing_api/world_location_news_article_presenter.rb

```diff
@@ -60,6 +60,16 @@ def image_available?
       item.images.any?
     end
 
+    def prepend_slash(path)
+      path.start_with?("/") ? path : path.prepend("/")
+    end
+
+    def image_path
+      image_path = presented_world_location_news_article.lead_image_path
+
+      Whitehall.public_asset_host + prepend_slash(image_path)
```

Comment:
Okedokey - I've changed the implementation in e402ee47c6ffe5c0b5fdd256003ee75debe57063 - how does that look @andrewgarner ?